Duplicati/Server/webroot/ngax/scripts/filters/parsetimestamp.js

```diff
@@ -14,3 +14,13 @@ backupApp.filter('parsetimestamp', function(AppUtils) {
       return AppUtils.toDisplayDateAndTime(value);
   }
 });
+
+backupApp.filter('parseForceActualDate', function(parsetimestampFilter, momentFilter) {
```

Comment:
Aesthetically, I'd like the `parseForceActualDate` filter to be named something more generic since, without the setting enabled, `parseForceActualDate` won't parse the actual date.

Reply:
Any suggestions?

Reply:
`parseDate`? Unless it overlaps with some existing filter